readme.md

```diff
@@ -72,6 +72,20 @@ Default: `false`
 
 Format the number using the [Binary Prefix](https://en.wikipedia.org/wiki/Binary_prefix) instead of the [SI Prefix](https://en.wikipedia.org/wiki/SI_Prefix). This can be useful for presenting memory amounts. However, this should not be used for presenting file sizes.
 
+##### digits
+
+Type: `number`\
+Default: `3`
+
+Number of significant digits to display when calculating bytes or bits.
+
+##### fixed
+
+Type: `boolean`\
+Default: `false`
+
+Format the number using fixed-point notation, rounded up.
```

Comment:
From the description, it's not clear to a user what this actually does. I think it should be explained in simpler terms.